product_variant_search_by_attribute/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
+# © 2016 David BEAL @ Akretion <david.beal@akretion.com>
+# © 2016 Raphaël REVERDY @ Akretion <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Product Variant Search by Attributes',
+    'summary': """Allows to search product variants matching several keywords
+                  in their attribute values.""",
+    'version': '8.0.0.0.1',
+    'category': 'Product',
+    'author': 'Akretion',
```

Comment:
I think the guideline is to add the 'Odoo Community Association (OCA)' author here to give a good visibility to the OCA on that crappy appstore.